CHANGELOG.md

```diff
@@ -1,3 +1,100 @@
+# v1.15.0 (2023-09-18)
+
+## Deprecation Notice
+
+The functionality to apply a hotpatch for log4j CVE-2021-44228 has been removed. The corresponding setting, `settings.oci-hooks.log4j-hotpatch-enabled`, is still available for backwards compatibility. However, it has no effect beyond printing a deprecation warning to the system logs. ([#3401])
+
+## OS Changes
+
+* Add kernel 6.1 ([#3121], [#3441])
+* Updated to kernel 5.10.192 and 5.15.128 ([#3441])
+* Enable Intel VMD driver for metal variants ([#3419])
+* Update admin and control containers ([#3368])
+* Update third party packages and dependencies ([#3362], [#3369], [#3330], [#3339], [#3456])
+* Updated to systemd 252 ([#3290])
+* Add Secure Boot to new variants ([#3097])
+* Add aws-iam-authenticator to metal variants ([#3357])
+* Add linux-firmware to metal variants ([#3296], [#3418])
+* Add support for XFS ([#3198])
+* Add `apiclient report` command including ([#3258]):
+  * Bottlerocket CIS benchmark report ([#2881])
+  * Kubernetes CIS benchmark report ([#3239])
+* Add resource-limit settings for OCI defaults ([#3206])
+* Use `systemd-networkd` and `systemd-resolved` instead of `wicked` for `aws-k8s-*`, `aws-ecs-2`, and `*-dev` variants ([#3134], [#3232], [#3266], [#3311], [#3394], [#3395], [#3451], [#3455])
+
+## Orchestrator Changes
+
+### ECS
+
+* Add `aws-ecs-2` variants ([#3273])
+* Add ECS Exec support ([#3075])
+* Add support for AppMesh ([#3267])
+
+### Kubernetes
+
+* Add Kubernetes 1.28 variants ([#3329])
+* Drop Kubernetes 1.22 variants ([#2988])
+* Update to Kubernetes 1.27.4 ([#3319])
+* Update to Kubernetes 1.26.7 ([#3320])
+* Update to Kubernetes 1.25.12 ([#3321])
+* Update to Kubernetes 1.24.16 ([#3322])
+* Add support for SeccompDefault setting for k8s 1.25+ ([#3334])
+
+## Build Changes
+
+* Upgrade to Bottlerocket SDK v0.34.1 ([#3445])
+* Use [Twoliter] to enable work on [out-of-tree builds]. All `tools` have moved to [Twoliter] ([#3342])
+* Only limit concurrency while building RPMs ([#3379], [#3429], [#3392], [#3342])
```

Comment:
This list of PRs doesn't seem right - should these all be twoliter related?

Reply:
Looks like an edit typo. Will fix.